template/default.tmpl

```diff
@@ -1,19 +1,17 @@
-{{ define "__subject" }}{{$dot := .}}[{{ .Status }}:{{ .Alerts | len }}] {{ range $k := .GroupLabelnames }}{{ index $dot.GroupLabels $k }} {{ end }}{{if gt (len .AlertCommonLabels) (len .GroupLabels) }}({{ range $k := .AlertCommonLabelnames }}{{ if eq "" (index $dot.GroupLabels $k) }}{{ index $dot.AlertCommonLabels $k }}{{ end }} {{ end }}){{ end }}{{ end }}
-
-{{ define "slack.default.fallback" }}
-{{ template "__subject" . }}
-{{ end }}
+{{ define "__alertmanager" }}AlertManager{{ end }}
+{{ define "__alertmanagerURL" }}{{ .ExternalURL }}{{ end }}
+{{ define "__subject" }}{{$dot := .}}[{{ .Status | upper }}:{{ .Alerts | firing | len }}] {{ range $i, $p := .GroupLabels | pairs }}{{ $p.Value }} {{ end }}{{if gt (len .CommonLabels) (len .GroupLabels) }}({{ range $i, $p := .CommonLabels | pairs }}{{ if eq "" (index $dot.GroupLabels $p.Name) }}{{ $p.Value }} {{ end }}{{ end }}){{ end }}{{ end }}
```

Comment:
The first depends on the other comment.
The second, I would consider this part of the documentation. If I don't care about order I have no reason to use that function in the first place.